### ISSUE TITLE
html inline style attributes corrupted

### ISSUE THREAD

**Issue by andystevenson** [GoogleChromeLabs/postcss-jit-props#12]
I was building a site with eleventy, open-props, postcss-jit-props and stumbled across this build problem.

The following inline style attribute 
```html
<section class='nested' style="background-image: url(/images/me.jpg)">
  <h2>{{title}}</h2>
</section>
```

gets transformed to 

```html
    <section class='nested' style=":root {}
background-image: url(/images/me.jpg)">
      <h2>is this image broken?</h2>
    </section>
```

I am assuming by ```postcss-jit-props```.

I have created a simple repo to reproduce the bug in ```https://github.com/andystevenson/weird-inline-style-build-bug```.

To reproduce run
```
npm install
npm run dev
```

You will observe 2 images (first one is unsplash)

<img width="362" alt="image" src="https://user-images.githubusercontent.com/1989620/158028108-21114eaa-2e96-4358-a6f7-8a6ee4aceb8d.png">

Then run 
```
npm run build
npm run preview
```

and observe the second image is missing because of the above issue.

<img width="360" alt="image" src="https://user-images.githubusercontent.com/1989620/158028202-4b095a41-4778-4c31-8e3d-cca57b88c882.png">

**Comment by andystevenson** [comment]
Hi Adam, I tried to have a look at the postcss-jit-code but wow! You need to know a lot about postcss API and objects to make any headway. I thought there might be an attribute on a Declartion object to know if it was an inline style? Similarly I was wondering if there was a way to know if the :root{} object was empty at the end of the transformation just to leave the original css alone?

**Comment by argyleink** [comment]
wow, definitely looks like a postcss-jit-props bug!
hm, it seems like postcss is initializing the plugin just for that inline style, which is why :root{} is created. then since no props are used, no props are added to the :root {} context. 
ok lol, 30m later, i've got a fix for it! thank you for the repo with repro, soo helpful.

**Comment by argyleink** [comment]
fixed is v1.0.5

**Comment by andystevenson** [comment]
Thanks Adam, especially for the quick fix!
